utils/nnn/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nnn
+PKG_VERSION:=2.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/jarun/nnn/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=0592c7cbcf2cf66cacac49e9204636480820b1bc74e4187dd7ee06945a6d07c5
+
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
```

Comment:
maybe add PKG_INSTALL and PKG_BUILD_PARALLEL?

Reply:
PKG_INSTALL is not used in many packages and I don't think it is needed for this package. I don't mind to add PKG_BUILD_PARALLEL.

Reply:
Added PKG_BUILD_PARALLEL.

Reply:
Not a specific package issues but someone should really try to push this again being 2019.  https://github.com/openwrt/openwrt/pull/1508